src/math/trig.c

```diff
@@ -168,7 +158,7 @@ int32_t is_scalar_cordic_acos(int32_t cosvalue, int16_t numiters)
 			y += ydshift;
 		} else {
 			sign = (((x > cosvalue) && (y >= 0)) ||
-				((x < cosvalue) && (y < 0))) ? 1 : -1;
+				((x < cosvalue) && (y < 0))) * 2 - 1;
```

Comment:
multiplication is faster than a ternary?

Reply:
Yes, a bit faster !!